frontend/plugins/frontline_ui/src/modules/forms/components/form-page/form-columns.tsx

```
@@ -0,0 +1,157 @@
+import { useNavigate } from "react-router";
+import { FormInstallScript } from "../FormInstallScript";
+import { IconCalendarEvent, IconEdit, IconTag, IconTrash, IconUser } from "@tabler/icons-react";
+import { useMutation } from "@apollo/client";
+import { FORM_REMOVE } from "@/forms/graphql/formMutations";
+import { MembersInline, SelectTags } from "ui-modules";
+
+export const actionColumn: ColumnDef<IForm> = {
+  accessorKey: 'actions',
+  id: 'actions',
+  cell: ({ cell }) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

ESLint

[error] Definition for rule 'react-hooks/rules-of-hooks' was not found.

(react-hooks/rules-of-hooks)